pom.xml

```diff
@@ -23,12 +23,11 @@
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <structured-logging.version>1.9.12</structured-logging.version>
         <junit-jupiter.version>5.5.2</junit-jupiter.version>
         <mockito.version>3.1.0</mockito.version>
         <kafka-models.version>1.0.19</kafka-models.version>
         <spring-kafka.version>2.8.0</spring-kafka.version>
```

Comment:
The latest version is 2.8.5, thinking maybe we should update to that version as we're updating the POM